app/includes/header.njk

```diff
@@ -23,9 +23,9 @@
   dataTestId: "header-banner",
   innerComponent: loginLogout({
     dataTestId: "login-logout-component",
-    username: username,
+    username: 'kk',
```

Comment:
just one `k` off a warning 👿

Reply:
actually why has this been changed?

Reply:
so I could see it was working - need to revert